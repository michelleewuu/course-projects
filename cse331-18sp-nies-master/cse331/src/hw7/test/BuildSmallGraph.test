# Test building a graph with data smallGraph.tsv

# Load smallGraph.tsv
LoadGraph smallGraph smallGraph.tsv

# Check if graph is correctly built
ListNodes smallGraph
ListChildren smallGraph NodeA
ListChildren smallGraph NodeB
ListChildren smallGraph NodeC