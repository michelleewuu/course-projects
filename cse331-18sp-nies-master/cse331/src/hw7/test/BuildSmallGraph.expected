# Test building a graph with data smallGraph.tsv

# Load smallGraph.tsv
loaded graph smallGraph

# Check if graph is correctly built
smallGraph contains: NodeA NodeB NodeC
the children of NodeA in smallGraph are: NodeB(1.000)
the children of NodeB in smallGraph are: NodeA(1.000)
the children of NodeC in smallGraph are: